pkgs/games/tome4/default.nix

```diff
@@ -1,21 +1,22 @@
-{stdenv, fetchurl, openal, libpng, libvorbis, mesa_glu, premake4, SDL2, SDL2_image, SDL2_ttf }:
+{stdenv, fetchurl, openal, libpng, libvorbis, mesa_glu, premake4, SDL2, SDL2_image, SDL2_ttf, bash }:
 
 stdenv.mkDerivation rec {
-  version = "1.4.6";
+  version = "1.4.9";
   name = "tome4-${version}";
   src = fetchurl {
     url = "http://te4.org/dl/t-engine/t-engine4-src-${version}.tar.bz2";
-    sha256 = "12pi2lw1k6l3p209nnkh4nfv3ppp8kpd6mkh1324c81z6mh6w4wg";
+    sha256 = "0c82m0g1ps64zghgdrp78m6bvfngcb75whhknqiailld7kz1g9xl";
   };
   buildInputs = [ mesa_glu openal libpng libvorbis SDL2 SDL2_ttf SDL2_image premake4 ];
   NIX_CFLAGS_COMPILE = [ "-I${SDL2_image}/include/SDL2" "-I${SDL2_ttf}/include/SDL2" ];
   preConfigure = ''
-    sed -e "s@/opt/SDL-2.0/include/SDL2@${SDL2}/include/SDL2@g" -e "s@/usr/include/GL@/run/opengl-driver/include@g" -i premake4.lua
+    sed -e "s@/opt/SDL-2.0/include/SDL2@${SDL2.dev}/include/SDL2@g" -e "s@/usr/include/GL@/run/opengl-driver/include@g" -i premake4.lua
     premake4 gmake
   '';
   makeFlags = [ "config=release" ];
   installPhase = ''
     install -Dm755 t-engine $out/opt/tome4/t-engine
+    echo "#!/{bash}/bin/sh" >> tome4
```

Comment:
You can also do `${stdenv.shell}` here, or simply use `/bin/sh` and rely on shebang patching to fixup the path.